pkg/netpol/eval/internal/k8s/baseline_admin_netpol.go

```diff
@@ -71,3 +71,37 @@ func (banp *BaselineAdminNetworkPolicy) GetIngressPolicyConns(src, dst Peer) (*P
 	}
 	return res, nil
 }
+
+func (banp *BaselineAdminNetworkPolicy) CheckEgressConnAllowed(dst Peer, protocol, port string) (res int, captured bool, err error) {
```

Comment:
missing documentation for these functions

Reply:
why does it need to return `int` and not just `bool` for whether allowed or not? why is `captured` needed?

Reply:
done